.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: Test
```

Comment:
Can you give this workflow a more specific name, e.g. tb_plugin_test.yml?

Reply:
Thanks, renamed.